packages/core/core/src/RequestTracker.js

```diff
@@ -1123,6 +1131,25 @@ export default class RequestTracker {
     await Promise.all(promises);
   }
 
+  writeNodeToCache(nodeId: NodeId) {
+    if (this.options.shouldDisableCache) {
+      return;
+    }
+    let node = this.graph.getNode(nodeId);
+    if (!node || node.type !== 'request') {
+      return;
+    }
+
+    let resultCacheKey = node.value.resultCacheKey;
+    if (resultCacheKey != null && node.value.result != null) {
+      this.options.cache.setLargeBlob(
+        resultCacheKey,
+        serialize(node.value.result),
```

Comment:
Does this slow down the build speed? If we're storing nodes in the cache with a heavy serialisation phase, we'll be blocking the event loop for a while. That might be ok if it occurs after the build is available in the dev server

Reply:
Yeah totally agree, I think we couldn't merge this in as in. Your approach with writing to the disk during idle time makes a lot more sense 😅